docs/rst/tutorials/aml_pipeline.rst

```diff
@@ -1,24 +1,452 @@
+    final unionModel size: 9884
+     + region  1 > 100
+     - region  1 > 900
+    Number of indicators 2930
+    Number of unique indicators 2930
+    Number of indicators 2930
+    Number of unique indicators 2930
+    <Preparing space class...>
+    <Calculating free traces...> (0.077s : c 0.075s - py 0.002s)
+    Number of indicators after selecting useful 1316
+    negative duple indicators 8
+    Final number of indicators: 56
+    <Calculating traces...> (0.049s : c 0.045s - py 0.004s)
+    - 1  <>--
+    + 2  + 4  + 1  + 1  + 2  + 1  + 1  + 2  + 3  + 1  + 2  + 1  + 1  + 1  + 2  --
+    --
+    --
+    From 26 to 26 (repetition)
+    Traces enforced with 26 atoms
+    From 1002 to 1002 (repetition)
+    <Traces of constants>
+    CrossAll time: 0.074s
+    From 10831 to 9977 (repetition)
+    G spectrum:
+      G 0 atoms 452
+      G 1 atoms 8
+      G 2 atoms 14
+      G 3 atoms 38
+      G 4 atoms 70
+      G 5 atoms 80
+      G 6 atoms 104
+      G 7 atoms 94
+      G 8 atoms 54
+      G 9 atoms 22
+      G 10 atoms 7
+      G 11 atoms 2
+      G 12 atoms 2
+    L spectrum:
+      L 1 atoms 452
+      L 2 atoms 8
+      L 3 atoms 14
+      L 4 atoms 28
+      L 5 atoms 46
+      L 6 atoms 38
+      L 7 atoms 22
+      L 8 atoms 25
+      L 9 atoms 18
+      L 10 atoms 20
+      L 11 atoms 17
+      L 12 atoms 19
+      L 13 atoms 20
+      L 14 atoms 19
+      L 15 atoms 7
+      L 16 atoms 17
+      L 17 atoms 17
+      L 18 atoms 21
+      L 19 atoms 18
+      L 20 atoms 15
+      L 21 atoms 19
+      L 22 atoms 8
+      L 23 atoms 11
+      L 24 atoms 9
+      L 25 atoms 8
+      L 26 atoms 9
+      L 27 atoms 5
+      L 28 atoms 6
+      L 29 atoms 4
+      L 30 atoms 3
+      L 31 atoms 3
+      L 32 atoms 3
+      L 33 atoms 2
+      L 34 atoms 3
+      L 35 atoms 1
+      L 36 atoms 2
+      L 37 atoms 3
+      L 39 atoms 1
+      L 40 atoms 1
+      L 41 atoms 2
+      L 42 atoms 1
+      L 44 atoms 2
+    fraction of negative indicators:  0.14285714285714285 , union model fraction:  15
+    BATCH: 948 batchSize( 100 , 900              ------------- master:  model size 9977
+
+    ... following iterations ...
+
+
+Postprocessing
+==============
+
+Here you can explore the atomization in the last model: `model.atomization`
+
+But also the cumulative model that has been built combining the atomizations across the whole training process: `batchLearner.unionModel`
+
+
+.. code-block:: python
+
+    print(f"{cGreen}Union model size:{cReset} {len(batchLearner.unionModel)}")
+    print(f"{cGreen}Size spectrum:{cReset}")
+    sc.printLSpectrum(batchLearner.unionModel)
+
+    print(f"{cGreen}Some random atoms:{cReset}")
+    import random
+    for at in random.choices(model.atomization, k=10):
+        print(at.ucs)
+
+    batchLearner.unionModel.sort(reverse=True,key=lambda at: len(at.ucs))
+    print(f"{cGreen}Largest atom{cReset}")
+    print(batchLearner.unionModel[0].ucs)
+
+
+This displays the following information::
```

Comment:
I'd explain more about this information, i.e. what can we deduce from it or how could we interpret it to check if everything looks good.